src/node.cc

```diff
@@ -1159,6 +1153,32 @@ Maybe<bool> ProcessEmitDeprecationWarning(Environment* env,
                                    deprecation_code);
 }
 
+static void OnMessage(Local<Message> message, Local<Value> error) {
+  Isolate* isolate = Isolate::GetCurrent();
+  switch (message->ErrorLevel()) {
+    case Isolate::MessageErrorLevel::kMessageWarning: {
+      Environment* env = Environment::GetCurrent(isolate);
+      if (!env) {
+        break;
+      }
+      v8::String::Utf8Value filename(isolate,
```

Comment:
Any particular reason for using `v8::String::Utf8Value` instead of `node::Utf8Value`?